src/App.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render } from '@testing-library/react';
+
+import { Provider } from 'react-redux';
+
+import App from './App';
+
+import store from './store';
+
+describe('App컴포넌트 관련 테스트', () => {
+  it('render로 App 화면을 그려지는 모습 확인 ', () => {
```

Comment:
테스트를 자세하게 작성해주셨군요ㅎㅎ 조금만 더 간결하게 작성해줘도 좋을 것 같아요~ 😅

이게 테스트인건 파일 네임을 통해 알 수 있죠.
```suggestion
describe('App', () => {
  it('레스토랑 정보를 입력하는 폼을 그린다 ', () => {
```
위와 같이 작성해주는건 어떤가요? 사용 설명서 측면에서 어떤걸 알려주고 싶은지 고민해보시면 좋을 것 같습니다. 😄

Reply:
넵 감사합니다.
사용 설명서 측면에서 고민해보도록 해보겠습니다 🔥